qemu/tests/ovs_qos.py

```diff
@@ -80,12 +80,12 @@ def get_throughout(netperf_server, server_vm, netperf_client,
                       logging.info)
         server_ip = server_vm.get_address()
         output = netperf_client.start(server_ip, client_options)
+        logging.info("netperf client output: %s" % output)
```

Comment:
We'd better not put the netperf output in the info log, maybe debug log is more suitable